languages/python/tests/test_polar.py

```diff
@@ -378,7 +378,7 @@ def test_parser_errors(polar):
 
 def test_runtime_errors(polar, query):
     rules = """
-    foo(a,b) := a in b;
+    foo(a,b) if a in b;
```

Comment:
I don't believe it's of much consequence, but I've stumbled upon the `polar_str` in traces -- does this need to be updated, too? https://github.com/osohq/oso/blob/ff8fc2d524d6c3b7eccb3be2935a32f4dae45237/polar/src/formatting.rs#L256

Reply:
Yep! Good catch.